java/examples/datasets/JavaDatasetExample.sh.in

```diff
@@ -155,10 +160,7 @@ CLEAN_LIBFILES_AND_BLDLIBDIR()
     INODE_SDIR=`$LS -i -d $SDIR | $AWK -F' ' '{print $1}'`
     INODE_DDIR=`$LS -i -d $BLDLIBDIR | $AWK -F' ' '{print $1}'`
     if [ "$INODE_SDIR" != "$INODE_DDIR" ]; then
-        for tstfile in $COPY_JARTESTFILES
-        do
-            $RM $BLDLIBDIR/tstfile
```

Comment:
All the files need to be deleted, not just the JAR files.  Plus this should have been $tstfile